RELEASE.md

```diff
@@ -1,3 +1,34 @@
+# Release 2.2.1
+
+This patch release includes a rollup to HEAD so as to support required features
```

Comment:
I would just omit this line entirely.  It could confuse users, and the rollup to HEAD is functionally the same as if we just cherrypicked a bunch of extra commits.